### PR TITLE
Middleware Unit Test

### PR DESCRIPTION
**Pull request by saraialvarezrosas**

### CODE REVIEW

File: searchobject/src/test/java/org/jalau/at18/searchobject/middleware/EmotionRecognitionMiddlewareTest.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
+package org.jalau.at18.searchobject.middleware;
+/**
+ * Copyright (c) 2022 Jala University.
+ *
+ * This software is the confidential and proprietary information of Jalasoft
```
Comment (rodri-98) [inline searchobject/src/test/java/org/jalau/at18/searchobject/middleware/EmotionRecognitionMiddlewareTest.java]: Typo property

File: searchobject/src/test/java/org/jalau/at18/searchobject/middleware/FaceDetectionControllerMiddlewareTest.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
+package org.jalau.at18.searchobject.middleware;
+/**
+ * Copyright (c) 2022 Jala University.
+ *
+ * This software is the confidential and proprietary information of Jalasoft
```
Comment (rodri-98) [inline searchobject/src/test/java/org/jalau/at18/searchobject/middleware/FaceDetectionControllerMiddlewareTest.java]: Typo property

File: searchobject/src/test/java/org/jalau/at18/searchobject/middleware/ProcessFrameControllerMiddlewareTest.java [hunk 3]
```diff
@@ -0,0 +1,93 @@
+package org.jalau.at18.searchobject.middleware;
+
+/**
+ * Copyright (c) 2022 Jala University.
+ *
+ * This software is the confidential and proprietary information of Jalasoft
```
Comment (rodri-98) [inline searchobject/src/test/java/org/jalau/at18/searchobject/middleware/ProcessFrameControllerMiddlewareTest.java]: Typo property

**Review by DaniMB95 — APPROVED** [review]

**Comment by DaniMB95** [comment]
please solve the conflicts

**Review by rodri-98 — APPROVED** [review]
Good job

**Pull request merged**